src/compute/servers.rs

```diff
@@ -325,15 +325,15 @@ impl Server {
 
     /// Stop the server, optionally wait for it to be powered off.
     pub async fn stop(&mut self) -> Result<ServerStatusWaiter<'_>> {
-        self.action(ServerAction::Stop).await?;
+        let _ = self.action(ServerAction::Stop).await?;
         Ok(ServerStatusWaiter {
             server: self,
             target: protocol::ServerStatus::ShutOff,
         })
     }
 
     /// Run an action on the server.
-    pub async fn action(&mut self, action: ServerAction) -> Result<()> {
+    pub async fn action(&mut self, action: ServerAction) -> Result<Option<serde_json::Value>> {
```

Comment:
I wonder if we can consider this a breaking change (because Rust will insist on `let _ = ...` by default). It's not an objection, just wondering how I should release rust-openstack after this.

Reply:
I'm also really unsure about using serde_json::Value in a public function. I'm trying to design the high-level API here to be type safe. Is it only for GetConsoleOutput? Maybe make separate methods for actions that return something?